spec.bs

```diff
@@ -2663,11 +2660,18 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|, a
   1. Let |igAd| be the [=interest group ad=] from |winner|'s [=generated bid/interest group=]'s
     [=interest group/ads=] whose [=interest group ad/render url=] is |winner|'s
     [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
-  1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-     [=map/exists=] and the result of [=query reporting ID k-anonymity count=]
-     given |winner|'s [=generated bid/interest group=] and |igAd| is true, then [=map/set=]
-     |browserSignals|["{{ReportingBrowserSignals/buyerAndSellerReportingId}}"] to |igAd|'s
-     [=interest group ad/buyer and seller reporting ID=].
+  1. If the |winner|'s [=generated bid/selected buyer and seller reporting ID=] is not null and the

```

Comment:
This isn't quite right. The way this reads right now, it says ...
```
If the selectedBuyerAndSellerReportingId is not null and it's k-anonymous, then set both fields on the reportResultBrowserSignals.
Otherwise (if the selectedBuyerAndSellerReportingId is null or it's not k-anonymous):
  If it's k-anonymous without the selectedBuyerAndSellerReportingId, then set only buyerAndSellerReportingId on the reportResultBrowserSignals.
```

Instead, this should read ...
```
If the selectedBuyerAndSellerReportingId is not null:
  If it's k-anonymous, then set both fields on the reportResultBrowserSignals.
Otherwise (if the selectedBuyerAndSellerReportingId is null):
  If it's k-anonymous , then set only buyerAndSellerReportingId on the reportResultBrowserSignals.
```

Reply:
Nice catch, just made a new commit for this